x-pack/legacy/plugins/watcher/public/services/watches/watches.js

```diff
@@ -4,23 +4,19 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
```

Comment:
While you are at this, maybe `.js` => `.ts`.

Reply:
Its a good idea, but there's the potential to get stuck on something and fail to merge this PR before I go on leave.